frontend/src/main/pages/CourseSearchPage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import CourseSearchForm from "main/components/CourseSearch/CourseSearchForm";
+
+import { useBackend } from "main/utils/useBackend";
+const CourseSearchPage = () => {
+
+    return (
+        <BasicLayout>
+            <h2>Course Search</h2>
+            <CourseSearchForm/>
+        </BasicLayout>
```

Comment:
Needs to implement the course table, should be included in new issue

Reply:
I structured this PR very poorly and as a result  I have to lump the incomplete page in with all my components, not really sure if it's fixable at this point

Reply:
I'm gonna approve it, but open a new issue with the other components so that Jeffrey/Nick's PR has a target issue

Reply:
Do you think modifying the requirements of issue #22 and #21 to be more specific to the components that need to be added would work, or should I just make a issues that specify the form and page?

Reply:
EIther works, just let the team know in slack